Conflicting RocksDB config option in Windows (lib linkage problem)

**Problem description**
When executed in Windows, Pravega Standalone exhibits a linkage error related to a RocksDB compression library:
```
Caused by: org.rocksdb.RocksDBException: Compression type Snappy is not linked with the binary.
```
This problem seems to impact only Windows environments. A related issue can be found here: https://github.com/facebook/rocksdb/issues/1266.

**Problem location**
RocksDB config.

**Suggestions for an improvement**
Remove the `optimizeLevelStyleCompaction()` config option, as it seems to be the reason for this problem.

--- a/segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/RocksDBCache.java
+++ b/segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/RocksDBCache.java
@@ -274,7 +274,6 @@ class RocksDBCache implements Cache {
                 .setMaxBackgroundFlushes(INTERNAL_ROCKSDB_PARALLELISM / 2)
                 .setMaxBackgroundJobs(INTERNAL_ROCKSDB_PARALLELISM)
                 .setCompactionStyle(CompactionStyle.LEVEL)
-                .optimizeLevelStyleCompaction()
                 .setMaxBackgroundCompactions(INTERNAL_ROCKSDB_PARALLELISM)
                 .setLevelCompactionDynamicLevelBytes(true);
 

